decomposed/extensions/MDDE_LDM/Profile/Model/Methods/mdde_InitializeExportExtension.xml

```diff
@@ -3,25 +3,44 @@
 <a:Name>mdde_InitializeExportExtension</a:Name>
 <a:CreationDate>1701423896</a:CreationDate>
 <a:Creator>WillemOtten</a:Creator>
+<a:Comment>This method is included in the extension to load the contents of templates that are defined in the optionally attached export extension.
+Since it is not possible to directly access the GTL templates in another extension via GTL, the contents of these templates is loaded in local extended attributes so it can be included in the XML export</a:Comment>
 <a:TargetCategory.Type>1</a:TargetCategory.Type>
 <a:MethodScript>Sub %Method%(obj)
-   &#39;Implement your method on &lt;obj&gt; here
-   &#39;Set success indicator to true
+   
    WriteInfo &quot;MDDE LDM Export extension initializion started&quot;
-   obj.setExtendedAttribute cMDDE_EXT_CODE &amp; &quot;.&quot; &amp; cMDDE_EXPORT_EXTENSION_INITIALIZATION_INDICATOR, True
    
    &#39;Set default template and attribute that apply to all object types
    Dim attributeTemplateName: attributeTemplateName = &quot;mddex_NamedObject_Attributes_Export_Template&quot;
    Dim attributeExtendedAttributeName: attributeExtendedAttributeName = &quot;mddex_NamedObject_Attributes_Export_Content&quot;
-   
+   Dim errorsFound: ErrorsFound = False   
+   Dim success: Success = False
```

Comment:
I think you should sync the variable casing here. Doesn't it throw an error now?

Reply:
Maybe use boolApplyTemplateSucceeded ?

Reply:
renamed